crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -561,6 +561,7 @@ en:
     upgrades:
       title: 'Upgrade'
       header: '%{product}: Upgrade from SUSE OpenStack Cloud 5'
+      repeat_after_conflict: 'Please repeat this step after resolving the conflicts'
```

Comment:
The dot at the end of sentence is missing. Also, I don't think "Please" is necessary here (but some English speaker should be judge of that).